src/Extensions/Constants/ShopUrls.php

```diff
@@ -87,6 +103,23 @@ public function orderPropertyFile($path)
         return $this->getShopUrl(RouteConfig::ORDER_PROPERTY_FILE, null, $path);
     }
 
+    public function tracking($orderId)
+    {
+        $lang = $this->sessionStorageService->getLang();
+        return $this->fromMemoryCache("tracking", function() use($orderId, $lang)
```

Comment:
Du solltest die Order-ID noch mit in den Cache-Key aufnehmen, falls mehrere Tracking-URLs für verschiedene Aufträge auf einer Seite geladen werden sollen.